authorization/http.go

```diff
@@ -75,7 +75,7 @@ func WithAuthorizers(authorizers map[string]rbac.Authorizer, permission rbac.Per
 
 			statusCode, ok, data := a.Authorize(subject, groups, permission, resource, tenant, tenantID, token)
 			if !ok {
-				w.WriteHeader(statusCode)
+				http.Error(w, "authorization declined", statusCode)
```

Comment:
I'm not convinced that saying "authorization declined" is any more informative to the user than the interpretation of a 403 as "forbidden".
But not a blocker

Reply:
The reason for this code is because as an outsider it took me a long time to realize where to start looking in the code to understand the reason for the 403.  The code generating the 403 forbidden was `w.WriteHeader(statusCode)` which has no "403" and no "forbidden".

The alternative I considered was a comment.  Which do you think is preferrable?

```
// Send 403 http.StatusForbidden
w.WriteHeader(statusCode)
```

Reply:
I think the second is better. Let's do that and merge this :)

Reply:
OK, done.  (The commit of the suggested alternative caused Github to believe I dismissed your reviews.)